provider/examples/resources/ios/ios_example.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  required_providers {
+    cdo = {
+      source = "hashicorp.com/CiscoDevnet/cdo"
+    }
+  }
+}
+
+provider "cdo" {
+  base_url  = "https://staging.dev.lockhart.io"
```

Comment:
Hey please take a look at asa_example.tf (unless my changes were overwritten), I wrote slightly better placeholder text than `FILL_ME`. Could you replicate that please?

Reply:
sure thanks